lessons/pydata/databases/index.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>Mohlo by se to postavit na datech z olympiád, která jsme použili na opakování (https://docs.google.com/document/d/1PvJM0tiy7kcT4NV0zM6uWnX6jLdlEiEc8P2fOs3xCok/edit). Jsou tam dvě tabulky, různé typy sloupců a možnost foreign key. Ale možná už je pozdě to měnit :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>